tests/fixtures/check_for_cran.yml

```diff
@@ -0,0 +1,186 @@
+http_interactions:
+- request:
+    method: get
+    uri: https://builder.r-hub.io/api/platform/list
```

Comment:
note that locally this means deactivating the RHUB_SERVER variable before recording.